src/main/scala/fi/oph/koski/opiskeluoikeus/PostgresOpiskeluoikeusRepository.scala

```diff
@@ -187,6 +188,29 @@ class PostgresOpiskeluoikeusRepository(
     }
   }
 
+  def checkValpasLainUlkopuolisiaPerusopetuksenVahvistettujaSuorituksia(oid: String)(implicit user: KoskiSpecificSession): Boolean = {
```

Comment:
Ymmärtääkseni opiskeluoikeuksia voi olla tallennettu  "master" tai "slave" oidille joten tämä pitäisi ottaa jotenkin tarkastelussa huomioon. Se voisi onnistua joinilla `henkilo` tauluun, jossa on kentät `oid` ja optionaalinen `master_oid`. Testiä varten `KoskiSpecificMockOppijat` löytyy linkitettyjä henkilöitä.